README.md

```diff
@@ -379,7 +379,7 @@ Comparison:
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/k0kubun/benchmark_driver.
+Bug reports and pull requests are welcome on GitHub at https://github.com/benchmark-driver/benchmark_driver.
```

Comment:
It should be https://github.com/benchmark-driver/benchmark-driver actually now :smile:  (although GitHub redirects fine)

Reply:
Haha, true :slightly_smiling_face:  https://github.com/benchmark-driver/benchmark-driver/commit/6ee9662953b29fd503ee2214ae4fbd08230a125d

Reply:
Ooops, totally missed it!